timescaledb/how-to-guides/page-index/page-index.js

```diff
@@ -444,11 +444,11 @@ module.exports = [
             excerpt: "High availability in TimescaleDB",
           },
           {
-            title: "Configure HA",
-            href: "configure-ha",
+            title: "Configure replication",
+            href: "configure-replication",
             tags: ["ha", "replicate", "timescaledb"],
             keywords: ["ha", "configure", "TimescaleDB"],
-            excerpt: "Configure high availability",
+            excerpt: "Configure replication",
```

Comment:
There are two configure replications here

Reply:
Ugh. I'll need to take a closer look.

Reply:
OK, that's definitely a hold over from the previous layout. I've deleted the extra entry now.